packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.controller.js

```diff
@@ -19,17 +21,38 @@ export default class PciTrainingJobsSubmitController {
     this.atInternet = atInternet;
   }
 
+  addHttpHeader() {
+    this.httpHeader.push({
+      added: false,
+      model: {},
+    });
+  }
+
+  onHttpHeaderAddBtnClick(index) {
+    this.httpHeader[index].added = true;
```

Comment:
This function adds a new empty row of inline adder. The data for `this.job.volumes` was being added on click of next step button.
Now i have added the logic inside `onHttpHeaderAddBtnClick` since it makes more sense to have it here.